pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -28,7 +28,9 @@ var info = capabilities.MustNewCapabilityInfo(
 	capabilities.CapabilityTypeConsensus,
 	"OCR3 consensus exposed as a capability.",
 	"v1.0.0",
-	nil,
+	&capabilities.DON{
+		ID: "00010203", // TODO: set real DON ID
+	},
```

Comment:
@bolekk This isn't how we should be setting the DON ID for local capabilities; this field is only intended for shimmed capabilities that are added via the syncer (at which point we'll know the the DON ID and its other settings.)

In my mind, we have two options for passing in the DON settings:
1. pass it through as part of the Execute request's metadata via the Workflow engine which already has access to the DON ID
2. pass it through when instantiating the OCR3 capability; and set it on the OCR3 capability job spec.

Let me think through which of these options is the best and implement it

Reply:
As I said on Slack, I think option 1 is best, assuming we want to pass through the WorklfowDonID, not the OCR3 Capability DON ID (I'm aware they're the same right now, but thinking ahead a bit)